conf/cassandra.yaml

```diff
@@ -158,6 +158,7 @@ authenticator:
 #  class_name : org.apache.cassandra.auth.MutualTlsAuthenticator
 #  parameters :
 #    validator_class_name: org.apache.cassandra.auth.SpiffeCertificateValidator
+#    max_certificate_age: 365d
```

Comment:
would be good to document that if not explicitly set, this is not validated.

Reply:
done, added a comment above